pkgs/top-level/aliases.nix

```diff
@@ -497,7 +497,7 @@ mapAliases ({
   morituri = whipper; # added 2018-09-13
   xfceUnstable = xfce4-14; # added 2019-09-17
   xfce4-14 = xfce;
-  xfce4-12 = xfce;
+  xfce4-12 = throw "xfce4-12 has been replaced by xfce4-14"; # added 2020-03-14
```

Comment:
Actually it was not replaced, the two had been coexisting, then `xfce4-12` was removed.

So it might be better to remove `xfce4-12` completely, instead of adding a `throw` message

Reply:
IMO the throw can help people know by what to replace `xfce4-12`, so we can only gain a small bit of user-friendliness by having it -- though not much, and I guess we can remove it soon(tm) in unstable, when most people will have made the change. And hopefully no one will think that `xfce4-14` actually contains the 4.12 version of xfce :smile: